lib/ClangImporter/ClangIncludePaths.cpp

```diff
@@ -227,19 +228,23 @@ getLibcFileMapping(ASTContext &ctx, StringRef modulemapFileName,
   SmallVector<std::pair<std::string, std::string>, 2> vfsMappings{
       {std::string(injectedModuleMapPath), std::string(actualModuleMapPath)}};
 
-  if (maybeHeaderFileName) {
-    // TODO: remove the SwiftGlibc.h header and reference all Glibc headers
-    // directly from the modulemap.
-    Path actualHeaderPath = actualModuleMapPath;
-    llvm::sys::path::remove_filename(actualHeaderPath);
-    llvm::sys::path::append(actualHeaderPath, maybeHeaderFileName.value());
+  auto addHeader = [&](std::optional<StringRef> maybeHeaderFileName) {
```

Comment:
What else can it be if it is not a header? The `maybe` here is about the fact that the file may not exist right?